dex/testing/loadbot/go.mod

```diff
@@ -1,6 +1,8 @@
 module decred.org/dcrdex/dex/testing/loadbot
 
-go 1.21
+go 1.21.10
+
+toolchain go1.23.0
```

Comment:
My `go mod tidy` seems to be adding the toolchain here. We don't want that though, correct?

Reply:
I guess one of the new imports is forcing this.

Reply:
You can confirm if you can build without it.

Reply:
I think it's fixed. It was @dev-warrior777 import but they fixed it.